internal/db/advisory_locks_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2025 Canonical.
+
+package db_test
+
+import (
+	"testing"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/frankban/quicktest/qtsuite"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/testutils/jimmtest"
+)
+
+type advisoryLocksSuite struct {
+	Database *db.Database
+}
+
+func (s *advisoryLocksSuite) Init(c *qt.C) {
+	db := &db.Database{
+		DB: jimmtest.PostgresDB(c, time.Now),
+	}
+	s.Database = db
+}
+
+func (s *advisoryLocksSuite) TestAdvisory_LockAndUnlock(c *qt.C) {
+	ctx := c.Context()
+
+	sqldb, err := s.Database.DB.DB()
+	c.Assert(err, qt.IsNil, qt.Commentf("Failed to get SQL DB connection"))
```

Comment:
if you are going to put `qt.Commentf` everywhere AND capitalize the first letter,  could we at least make it a full sentence and end with a full stop (".")? here and everywhere else?